library/kotlin/io/envoyproxy/envoymobile/HeadersContainer.kt

```diff
@@ -12,14 +12,27 @@ open class HeadersContainer {
   /**
    * Represents a header name together with all of its values.
    * It preserves the original casing of the header name.
+   * 
+   * @param name  The name of the header. Its casing is preserved.
+   * @param value The value associated with a given header.
    */
   data class Header(val name: String, var value: MutableList<String>) {
     constructor(name: String) : this(name, mutableListOf())
 
+    /**
+     * Add values.
+     * 
+     * @param value The list of values to add.
+     */
     fun add(value: List<String>) {
```

Comment:
That's my intuition too but since `HeadersBuilder` use `value` I wanted to stay consistent. Maybe there is actually a difference between `HeadersContainer` and `HeaderBuilder` and it's OK for them to use different names. Let me update.